Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM maven:3.8.1-jdk-8 as builder
+
+COPY pom.xml /tmp/pom.xml
+COPY src /tmp/src
+RUN mvn -f /tmp/pom.xml -s /usr/share/maven/ref/settings-docker.xml clean package
+
+FROM openjdk:8
```

Comment:
Haven't tested it, but wouldn't a smaller base image like openjdk:8-jre-slim be enough to run it ?

Reply:
:+1: Switched to slim images for Maven as well as for the runtime.